src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -445,6 +447,12 @@ private IDataView GetShuffledData(string path)
                         statisticsCallback(metrics);
                     }
                 }
+
+                //Early stopping check
+                if (options.EarlyStopper != null)
+                {
+                    earlyStop = options.EarlyStopper.ShouldStop(metrics.Train);
+                }
```

Comment:
why not just break out? it will save you a variable

Reply:
done

---
In reply to: [330316698](https://github.com/dotnet/machinelearning/pull/4237#discussion_r330316698) [](ancestors = 330316698)